PR: https://github.com/odoo/odoo/pull/

From: c91d44aef506c36b6f9a8ffcfd1443718a6134ed
From: Kinjal Mehta
Date: 2018-01-04 12:26:54

Structural Changes: 6
Total Changes: 6

[IMP] hr_attendance,hr_contract,hr_expense_check,hr_payroll,hr_recruitment: changed duplicate field label in the same model

================================= pseudo patch: =================================

--- a/addons/hr_recruitment/models/hr_job.py
+++ b/addons/hr_recruitment/models/hr_job.py
@@ -15,15 +15,15 @@ class Job(models.Model):
         'res.partner', "Job Location", default=_default_address_id,
         help="Address where employees are working")
     application_ids = fields.One2many('hr.applicant', 'job_id', "Applications")
-    application_count = fields.Integer(compute='_compute_application_count', string="Applications")
+    application_count = fields.Integer(compute='_compute_application_count', string="# of Applications")
     manager_id = fields.Many2one(
         'hr.employee', related='department_id.manager_id', string="Department Manager",
         readonly=True, store=True)
     user_id = fields.Many2one('res.users', "Recruitment Responsible", track_visibility='onchange')
     hr_responsible_id = fields.Many2one('res.users', "HR Responsible", track_visibility='onchange',
         help="Person responsible of validating the employee's contracts.")
-    document_ids = fields.One2many('ir.attachment', compute='_compute_document_ids', string="Applications")
-    documents_count = fields.Integer(compute='_compute_document_ids', string="Documents")
+    document_ids = fields.One2many('ir.attachment', compute='_compute_document_ids', string="Documents")
+    documents_count = fields.Integer(compute='_compute_document_ids', string="# of Documents")
     alias_id = fields.Many2one(
         'mail.alias', "Alias", ondelete="restrict", required=True,
         help="Email alias for this job position. New emails will automatically create new applicants for this job position.")
